rocAL/rocAL/source/randombboxcrop_reader.cpp

```diff
@@ -111,15 +112,16 @@ pCropCord RandomBBoxCropReader::get_crop_cord(const std::string &image_name)
     return it->second;
 }
 
-void RandomBBoxCropReader::add(std::string image_name, BoundingBoxCord crop_box)
+int RandomBBoxCropReader::add(std::string image_name, BoundingBoxCord crop_box)
```

Comment:
This implementation can't update the crop is there is already a crop exists. We need to change the crop box update when loading images.